server/src/ab-tests.js

```diff
@@ -39,7 +49,8 @@ let allTests = {
     version: 1,
     exclude: ["highlightButtonOnInstall", "autoOpenSharePanel"],
     options: [
-      // Note no one will end up in control in this example:
+      // Note no one will end up in exclude in this example (but 50% will still
+      // be control):
```

Comment:
@ianb If I'm understanding this correctly, since 10% of the population is in the `autoOpenSharePanel` test, and another 10% is in the `highlightButtonOnInstall` test, and people in those tests are excluded from this `myShotsDisplay` test, then, 50% of the remaining total 80% of users will be in the control group in this test, while the other half will be 9:1 split between the 'intropopup' and 'blink' tests. Does that seem correct?